src/examples/turtlebot4/README.md

```diff
@@ -0,0 +1,138 @@
+# Turtlebot4 tutorial
+
+The following tutorial will be presented during [ROSCon 2024](https://roscon.ros.org/2024/)
+talk about RAI.
+
+## Recording
+
+TBD
+
+## Step by step instructions
+
+1. O3DE simulation setup
+
+   1. Download turtlebot4 simulation binary matching your platform from :
+
+      - Ubuntu 22.04 & ros2 humble: [link](https://robotecai-my.sharepoint.com/:u:/g/personal/bartlomiej_boczek_robotec_ai/EZLmGtPNgl9Kiu4royJJnVgB5tjS2Vze0myXDyVJtNcnRw?e=L42Z4z)
+      - Ubuntu 24.04 & ros2 jazzy: [link](https://robotecai-my.sharepoint.com/:u:/g/personal/bartlomiej_boczek_robotec_ai/Ebi0O3eWAtRAl3GL8LuEBqIB0-5fnHDCm7yOtcTucpNfzA?e=G6eJKP)
+
+      > **NOTE** If you would like to make changes to the simulation and create your
+      > own binary please follow this [README.md](https://github.com/RobotecAI/ROSCon2024Tutorial/README.md)
+
+   2. Unzip it:
+
+      ```bash
+      cd rai
+      # for Ubuntu 22.04 jammy and ros2 humlbe
+      unzip -d src/examples/turtlebot4/simulation Turtlebot4_jammyhumble_0.0.1.zip
+
+      # for Ubuntu 24.04 noble and ros2 jazzy
+      unzip -d src/examples/turtlebot4/simulation Turtlebot4_noblejazzy_0.0.1.zip
+      ```
+
+2. Clone and install [rai](https://github.com/RobotecAI/rai)
+
+   1. Follow setps from [1. Setting up the workspace](https://github.com/RobotecAI/rai?tab=readme-ov-file#1-setting-up-the-workspace)
+   2. Then follow steps from [2. Build the project](https://github.com/RobotecAI/rai?tab=readme-ov-file#2-build-the-project)
+
+3. Setup your [OpenAI API](https://openai.com/index/openai-api/). Make sure you have set `OPENAI_API_KEY` variable.
```

Comment:
Instead, link to RAI vendor-agnostic setup. You can summarize (e.g. follow steps 1-3 of RAI)

Reply:
unfortunately this tutorial will not work well with Ollama, because of lack of vision-function calling model..